pkg/rsync/sync.go

```diff
@@ -74,14 +74,17 @@ func (r Rsync) SyncData(source string, target string, excludes ...string) error
 
 // MirrorData rsync's source folder contents to a target folder content, in contrast, to SyncData this
 // method adds the --delete flag which forces the deletion of files in target that are missing in source.
-func (r Rsync) MirrorData(source string, target string, excludes ...string) error {
+func (r Rsync) MirrorData(source string, target string, excludes []string, deleteExcludes []string) error {
 	flags := r.flags
 	if !slices.Contains(flags, "--delete") {
 		flags = append(flags, "--delete")
 	}
 	for _, e := range excludes {
 		flags = append(flags, fmt.Sprintf("--exclude=%s", e))
 	}
+	for _, e := range deleteExcludes {
+		flags = append(flags, fmt.Sprintf("--filter=protect %s", e))
```

Comment:
```suggestion
		flags = append(flags, fmt.Sprintf("--filter='protect %s'", e))
```

I'm not sure this actually works (untested) but at least the man page says it should be quoted

Reply:
I guess man page says that because it assumes to be called within a shell. This is not the case here, the space here is not handled as a separator by any interpreter. In fact I tried with quotes at first and had to remove them. See the same applies for the SLELinux extended attributes filter.

Also, unit tests are actually testing this behavior.